.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
+about: Report a bug encountered with Jenkins X docs
+labels:
+- area/docs
+- kind/bug
```

Comment:
Need to see if these labels for the bug report issue are correct.

Reply:
Remove the area/docs, not required imo. Every issue opened here would be area/docs.